### ISSUE TITLE
AwsSignatureHelper.getAmazonRegion() method does not count with https protocol

### ISSUE THREAD

**Issue by pe-pan** [CloudSlang/cs-content#1061]
This is very much related to the issue #994. I was trying to run [describe_instance.sl flow](https://github.com/CloudSlang/cs-content/blob/master/content/io/cloudslang/amazon/aws/ec2/instances/describe_instances.sl) with non-default region and it was failing on the very same error as in the issue #994 (_Credential must have exactly 5 slash-delimited elements, e.g. keyid/date/region/service/term, got 'SecretAmazonKey/ps://ec2/ec2/aws4_request,'_).

After some more detailed scrutation I realized that it's actually the method [AwsSignatureHelper.getAmazonRegion()](https://github.com/CloudSlang/cs-actions/blob/master/cs-amazon/src/main/java/io/cloudslang/content/amazon/services/helpers/AwsSignatureHelper.java) is incorrectly implemented as it contains this piece of code:
`endpoint = endpoint.substring(3);`

In other words, it counts that the endpoint contains the region from the forth character (as the very first three characters are either _ec2._ or _s3._). The method does not count with the fact that the endpoint actually has to start with "https://" protocol.

Also, from [the list of possible AWS API endpoints](https://docs.aws.amazon.com/general/latest/gr/rande.html), one can see that even AWS API name can contain hyphen; thus hyphen character cannot be used to distinguish the region part.
